.github/workflows/tests.yml

```diff
@@ -222,3 +222,71 @@ jobs:
 
       - name: "Tests"
         run: "make tests"
+
+  mutation-testing:
```

Comment:
I want this to run only in pull requests. Can be achieved with `if: github.event_name == 'pull_request'`